doc/visual-programming/source/widgets/visualize/pythagoreanforest.rst

```diff
@@ -0,0 +1,67 @@
+Pythagorean Forest
+==================
+
+.. figure:: icons/pythagorean-forest.png
+
+Pythagorean forest for visualising random forests.
+
+Signals
+-------
+
+**Inputs**:
+
+-  **Random Forest**
+
+Classification / regression tree models as random forest.
```

Comment:
It's done documentation-wide so I'm not changing it in this PR. Could be fixed in a subsequent PR that addresses indentation only.